static/app/components/events/interfaces/frame/contextLine.tsx

```diff
@@ -122,7 +122,9 @@ const StyledLi = styled('li')`
 `;
 
 // TODO(scttcper): The parent component should be a grid, currently has too many other children
+// If the grid was higher up we wouldn't have to hardcode the line number width and could fit larger line numbers
+const lineNumberWidth = '58px';
 const LineContent = styled('div')`
   display: grid;
-  grid-template-columns: 58px 1fr;
+  grid-template-columns: ${lineNumberWidth} calc(100% - ${lineNumberWidth} - ${space(2)});
```

Comment:
would it be possible to remove the magic number in `space(2)` and explain what its purpose is?